### ISSUE TITLE
Uso de yield from, está prohibido?

### ISSUE THREAD

**Issue by masoto22** [IIC2233/Syllabus#412]
### Prerrequisitos
(Marcar colocando una `X` entre los corchetes los ítems que ya hiciste, así: `"[X]"`)

* [X ] Leí las reglas del foro (https://github.com/IIC2233/Syllabus/wiki/6.-Reglas-del-foro-(issues)).
* [X] Busqué en las *issues* si ya preguntaron mi duda y no encontré nada parecido (https://github.com/IIC2233/Syllabus/issues/).
* [X] Revisé el compilado de dudas de la tarea y no encontré una *issue* similar a la mía ([click aquí para ir al compilado](/IIC2233/Syllabus/issues/396)).
* [X] Mi duda no se trata sobre una librería, *built-in* o mala práctica, ya que eso se pregunta en la *issue* creada para ello ([click aquí para ir a esa _issue_](/IIC2233/Syllabus/issues/395)).
* [X] Mi duda no se trata de un tema administrativo o personal, ya que en ese caso debo contactar a mi profe, al ayudante de Bienestar o al correo del curso (coordinacion.iic2233@gmail.com).
* [X] Utilizaré un título descriptivo y llenaré correctamente esta plantilla.
* [X] De ser necesario, colocaré código simple que permita explicar mi problema o duda, ya que compartir código de esta evaluación puede ser considerado una falta a la ética.

### Duda

Hola, quisiera saber si el uso de "yield from" está permitido para la sección donde se trabaja con generadores en la tarea? 

Gracias!

**Comment by victoriaatria** [comment]
Hola @masoto22 ! Podrías dar un ejemplo de cómo quieres utilizar `yield from` para entender mejor el tipo de uso al que te refieres?

**Comment by masoto22** [comment]
Sí! Quiero usarla recursivamente, en las funciones relacionadas a los generadores, donde si es = el genero o = la edad, retorna un generador con la pelicula, o persona. Entonces para hacer yield de todas las pelis/personas es que quisiera usar yield from(para tener todos los elementos del generador que entrega cada función) :)

**Comment by masoto22** [comment]
@victoriaatria No sé si se entendió bien a lo que me refería 🥹🥲
